plugin/pkg/admission/noderestriction/admission.go

```diff
@@ -113,6 +116,14 @@ func (c *nodePlugin) Admit(a admission.Attributes) error {
 	case nodeResource:
 		return c.admitNode(nodeName, a)
 
+	case pvcResource:
```

Comment:
I expected to see tests added to test/integration/auth/node_test.go exercising the authorizer/admission bits, in combination with the feature flag

* feature flag off, node gets forbidden for get/update/patch of unrelated pvc/status
* feature flag off, node gets forbidden for get/update/patch of related pvc/status
* feature flag on, node gets forbidden for get/update/patch of unrelated pvc/status
* feature flag on, node gets forbidden for get/update/patch of related pvc/status that changes non-whitelisted field
* feature flag on, node gets ok for get/update/patch of related pvc/status that changes whitelisted field

Reply:
Added integration tests